Sorting/selection_sort/selection_sort.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+
+using namespace std;
+
+void swapElement(int *m, int *n)  
+{  
+    int temp = *m;  
+    *m = *n;  
+    *n = temp;  
+}  
+  
+void selectionSort(int arr[], int n)  
+{  
+    int i, j, minVal;  
+  
+    // Loop unsorted Array values 
+    for (i = 0; i < n-1; i++)  
+    {  
+        // Find the lowest value  
+        minVal = i;  
+        for (j = i+1; j < n; j++)  
+        if (arr[j] < arr[minVal])  
+            minVal = j;  
+  
+        // Swap the minimum value element with the first available value at array index  
+        swapElement(&arr[minVal], &arr[i]);  
+    }  
+}  
+  
+void printArray(int arr[], int size)  
+{  
+    int i;  
+    for (i=0; i < size; i++)  
+        cout << arr[i] << " ";  
+    cout << endl;  
+}  
+  
+/*
+* Main Function
+* select dynamic array elements
+*/
+
+int main()
+{
+    int Array[6];
+    cout << "\nEnter any 6 Numbers for Unsorted Array : ";
+
```

Comment:
Can we take even the size of the array as input? 

That way it would be in the same format as standard competitive programming problems.

Reply:
Yes, Good point @nikochiko. I'll add the support to input array size as well.